### ISSUE TITLE
Create document outlining fiscal policy of this project

### ISSUE THREAD

**Issue by paymog** [MycroCoin/mycro#4]
We need to create a document outlining the fiscal policy of this project so that new contributors can get a sense of how much their work might be worth.

**Comment by Peddle** [comment]
Yeah. We're currently missing a lot of documentation. A link to our temporary centralized ledger would also likely be good
